### DISCUSSION TITLE
Add image in option's label

### DISCUSSION THREAD

**Discussion by quocuyvan** [sa-si-dev/virtual-select#41] [Q&A] [answered]
Can I add image before option's label?

**Comment by sa-si-dev** [comment]
As of now, only string is allowed to use as label. Will add support to use images in the future release.

**Comment by sa-si-dev** [comment, marked as answer]
@quocuyvan image support is available in the [release v1.0.11](https://github.com/sa-si-dev/virtual-select/releases/tag/v1.0.11) ([more details](https://sa-si-dev.github.io/virtual-select/#/examples?id=add-imageicon))

**Reply by MuhammedAlkhayyat** [reply]
selectedLabelRenderer not working with my VertualSelect, any ideas?
